projects/packages/my-jetpack/_inc/utils/prevent-widows.ts

```diff
@@ -0,0 +1,9 @@
+type PreventWidows = ( text: string ) => string;
+
+const preventWidows: PreventWidows = text => {
```

Comment:
Great idea about the idea of moving that function to this codebase!

Do you think it would be worth getting inspiration from this implementation? https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/formatting/prevent-widows.js

Reply:
I guess that type of implementation is more tested and widely used than the one I have here 😅 

I'll go ahead and change it to be the same

Reply:
I removed the cases where it is not a string since we don't use the same `translate` function in the plugin but the implementation is better nonetheless 😄